include/swift/AST/DiagnosticsSema.def

```diff
@@ -661,8 +661,12 @@ ERROR(init_not_instance_member,none,
       "a new object of the same dynamic type", ())
 ERROR(super_initializer_not_in_initializer,none,
       "'super.init' cannot be called outside of an initializer", ())
+
 WARNING(isa_is_always_true,none, "'%0' test is always true",
         (StringRef))
+WARNING(isa_is_foreign_check,none,
+      "'is' test always succeeds because %0 is a CoreFoundation type",
```

Comment:
> Phrasing nitpick: "Core Foundation", not "CoreFoundation".
> ...
> I'm also still not happy with "succeeds"

Agree.  Will fix.

> because it's not going to do a test at all.

This is styled on the language of the warning above.  I can change both if that is what is required.